### DISCUSSION TITLE
Cannot load a page from MDN

### DISCUSSION THREAD

**Discussion by worikgh** [mdn/mdn-community#276] [Content] [answered]
[This \<img> documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img)

I see a flash of content then a blank page.  I can load https://developer.mozilla.org/en-US/

If I start there and navigate in looking for markup documentation https://developer.mozilla.org/en-US/docs/Web/HTML is the first to fail.

I get there from https://developer.mozilla.org/en-US/docs/Learn/HTML

Firefox: 105.0.3 (64-bit)

uname -a
Linux munroe 5.4.0-128-generic #144-Ubuntu SMP Tue Sep 20 11:00:04 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

I am getting four errors in the console, how do I copy those?

I should list my extensions.  I am unsure how.

Very frustrating.  MDN is such a valuable resource.  I do very little web programming and it is my goto for tag syntax

**Comment by bsmth** [comment, marked as answer]
Hi @worikgh sorry to hear you are having trouble. We don't have any other reports of users with this issue which could indicate that it's something specific to your machine.

Are there any other websites that you experience this behavior? 

Can you try updating Firefox to the latest version?

>I should list my extensions

Disabling extensions would help narrow the cause down.

**Comment by worikgh** [comment]
Working now.   Mēh!

**Reply by bsmth** [reply]
Glad it's resolved! 🙌🏻
